script/web_discovery_project.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+from lib.config import PLATFORM, SOURCE_ROOT, enable_verbose_mode
+from lib.util import execute_stdout
+
+WEB_DISCOVERY_DIR = os.path.join(SOURCE_ROOT, 'vendor', 'web-discovery-project')
+
+NPM = 'npm'
+if PLATFORM in ['win32', 'cygwin']:
+    NPM += '.cmd'
+
+
+def main():
+    os.chdir(WEB_DISCOVERY_DIR)
+    args = parse_args()
+
+    if args.verbose:
+        enable_verbose_mode()
+    if args.install:
```

Comment:
for both of these commands `env = os.environ.copy()` and pass `env` to `execute_stdout`

Reply:
not sure why we're copying, but that's what we do everywhere else

Reply:
np. fixed in https://github.com/brave/brave-core/pull/9381/commits/6d5ba142250d01766ddfef63479775add99ae346

Reply:
it's probably some extra safety in case one of these methods mutates `env`.